.github/workflows/check-subpackages.yml

```diff
@@ -6,12 +6,16 @@ on:
 jobs:
   build-subpackages:
     runs-on: ubuntu-latest
```

Comment:
Why runs on ubuntu and not on fedora?

Reply:
since it is githubactions, which run only on ubuntu IIRC
this is why we use container in it 
see this:
    container:
      image: fedora:latest  # Use Fedora as the container image